index.js

```diff
@@ -1,15 +1,19 @@
 'use strict';
 var through = require('through2');
 
-module.exports = function () {
+module.exports = function (reversions) {
+	reversions = reversions || 1;
```

Comment:
`reversions = typeof reversions === 'number' ? reversions : 1;` as users might want to explicitly pass in `0` for some reason.